File: KotlinExample/app/src/main/java/fi/qvik/kotlinexample/SecondView.kt
package fi.qvik.kotlinexample

import androidx.compose.foundation.layout.Column
import androidx.compose.material3.Button
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable

@Composable
fun SecondView(name: String,
               goBack: () -> Unit) {
    Column {
        Text(text = "Second")
        Button(onClick = { goBack() }) {
            Text(text = "Back")
            
        }
    }
}

File: SwiftExample/SwiftExample/SecondView.swift
//
//  SecondView.swift
//  SwiftExample
//
//  Created by Matias Hätönen on 6.5.2024.
//

import Foundation
import SwiftUI

struct SecondView: View {
    var name: String
    
    var goBack: () -> Void
    var body: some View {
        VStack {
            Text("second")
            Button("Back") {
                goBack()
            }
        }
    }
}
